sdks/python/apache_beam/typehints/decorators_test.py

```diff
@@ -369,9 +369,7 @@ def test_disable_type_annotations(self):
     decorators.disable_type_annotations()
     self.assertTrue(decorators._disable_from_callable)
 
-
-class DecoratorsTest(unittest.TestCase):
```

Comment:
Added the content of the last two `DecoratorsTests` to the first one because its code wasn't being executed and also were throwing a `function-redefined` warning.